website/src/content/concepts/integrate-with-your-own-api.mdx

```diff
@@ -251,6 +251,131 @@ The setup described above is not needed when using the `@commercetools-backend/e
 
 </Info>
 
+# Local development of a Custom Application with external API
+
+Developing a Custom Application locally, together with an external API, has some limitations.
+For example, using the `forward-to` endpoint of the Merchant Center API Gateway with a target URL pointing to `http` is not allowed. Only `https` targets are allowed.
+Furthermore, the Merchant Center API Gateway cannot proxy a request to `localhost`.
+
+The most straighforward solution is using the production URL of the external API, which may be inconvenient in the development processs.
+
+Therefore, some kind of local tunnel solution can be used to develop both Custom Application and external API together, locally.
+
+## ngrok
+
+One of possible solutions is using the [ngrok](https://ngrok.com/) service. `ngrok` is a reverse proxy allowing to establish an encrypted tunnel between the `ngrok` agent application running locally and the upstream service.
+
+Please follow the [instructions](https://ngrok.com/download) to sign up and download the agent application. Once installed, start the agent application by running:
+```console
+ngrok http <desired_port>
+```
+where `<desired_port>` is the port bound to your server application (for example `6000`).
+
+The exposed external URL of the created tunnel:
+![ngrok agent](/images/ngrok.png) 
+can be further passed to the session middleware verifying the JWT audience claim.
+```js highlightLines="10"
+const express = require("express");
+const {
+  createSessionMiddleware,
+  CLOUD_IDENTIFIERS,
+} = require("@commercetools-backend/express");
+
+const app = express();
+app.use(
+  createSessionMiddleware({
+    audience: "https://33ea-87-183-162-9.eu.ngrok.io",
+    issuer: CLOUD_IDENTIFIERS.GCP_EU,
+  })
+);
+app.post("/", (req, res) => {
+  res.send("It works 🙌");
+});
+
+app.listen(6000, () => {
+  console.log("Running on port 6000.");
+});
+
+module.exports = app;
+```
+
+To verify the tunneling service, configure the additonal environment variable `echoServerApiUrl` in your Custom Application:
+```js title="custom-application-config.mjs" highlightLines="4"
+const config = {
+  // ...
+  additionalEnv: {
+    echoServerApiUrl: "https://33ea-87-183-162-9.eu.ngrok.io",
+  },
+```
+and use it in requests sent to the `/proxy/forward-to` endpoint:
+
+```jsx title="echo-server.jsx" highlightLines="8-10,12,19"
```

Comment:
This code snippet can also be omitted I think, as the usage of `/proxy/forward-to` is already documented elsewhere.

The important thing here is just to communicate to use a secure tunnel instead of the production URL, the logic in the code then stays the same.